packages/Button/src/button.scss

```diff
@@ -12,7 +12,7 @@
   --ids-btn-padding: #{tokens.$space-3};
   --ids-btn-padding-mobile: 1.2rem;
   --ids-btn-border-radius: #{tokens.$border-radius-sm};
-  --ids-btn-icon-color: currencolor;
+  --ids-btn-icon-color: currentColor;
   --ids-btn-icon-margin: #{tokens.$space-1};
   --ids-btn-z-index: #{tokens.$z-index-10};
   --ids-btn-focus-position: -0.3rem;
```

Comment:
`--ids-btn-z-index`, `--ids-btn-focus-position` and `--ids-btn-shadow` are not used anymore and can be deleted